espressif/idf-eclipse-plugin
# IEP-685: Wokwi esp simulator integration in Eclipse plugin

## Description

Wokwi esp32 simulator integration in Eclipse plugin

Fixes # ([IEP-685](https://jira.espressif.com:8443/browse/IEP-685))

## Type of change
- New feature (non-breaking change which adds functionality)

## How has this been tested?

- Install wokwi-server as mentioned [here](https://github.com/MabezDev/wokwi-server/) 
- Configure WOKWI_SERVER_PATH in the Eclipse CDT build environment variables with the wokwi-server executable path
- Create a new launch configuration with the "Wokwi Simulator" from the list
- Choose a project and add the project id of the wokwi project if required
- Click Finish to save the changes
- From the toolbar, click on the Launch button to launch the wokwi simulator
- wokwi simulator will be launched in the external browser
- Observe the serial monitor output in the Eclipse CDT build console

**Test Configuration**:
* ESP-IDF Version: 5.0
* OS (Windows,Linux and macOS):macOS

## Dependent components impacted by this PR:

- Launch

## Checklist
